docs/experiments-plus/getting-group.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Getting the Group
+sidebar_label:Getting the Group
+slug: /experiments-plus/getting-the-group
+---
+
+One of the most common questions we get is "How do I check the experiment group in code?"  Checking the experiment group _in code_ is actually an anti-pattern.
```

Comment:
Do we want to mention "one of the most common questions"? I feel that give people the sense that we decided to not do what a lot of people want, or do things in a weirdly different way.

Reply:
We can say something along the lines like "Often times new users have the misconception that they need to check which group a user is in for a given experiment in code, but that's actually not needed with Statsig and limits your flexibilities"